### PR TITLE
Add Xena portal integrations (#1345)

### PR DESCRIPTION
**Pull request by danielsotirhos**

### CODE REVIEW

File: src/azul/project/hca/__init__.py [hunk 1]
```diff
@@ -473,5 +473,171 @@ def portal_integrations_db(self) -> JSON:
...
                 ],
+            },
+            {
+                "portal_id": "2e05f611-16fb-4bf3-b860-aa500f0256de",
+                "portal_name": "Xena",
+                "portal_icon": "https://xenabrowser.net/03340e094d1f3edc51bc3d1a2a589b65.png",
+                "contact_email": "",
+                "organization_name": "UCSC",
+                "portal_description": "",
+                "integrations": [
```
Comment (hannes-ucsc) [inline src/azul/project/hca/__init__.py]: Time to switch to comprehensions, Daniel.

File: src/azul/project/hca/__init__.py [hunk 2]
```diff
@@ -473,5 +474,64 @@ def portal_integrations_db(self) -> JSON:
...
+                        "integration_type": "get",
+                        "entity_type": "project",
+                        "title": title,
+                        "entity_ids": {
+                            "staging": [],
+                            "integration": [prod_uuid],
+                            "prod": [prod_uuid]
+                        },
+                        "portal_url": "https://singlecell.xenabrowser.net/datapages/?cohort=" + quote(title)
+                    } for integration_id, title, prod_uuid in (
```
Comment (hannes-ucsc) [inline src/azul/project/hca/__init__.py]: 1) title should come last
2) rename prod_uuid to project_uuid
3) one tuple per line, and suppress warning

See diff

**Comment by danielsotirhos** [comment]
I apologize and take ownership for not realizing `JSON` was the wrong type in this case due to the return value not being a dict but a list of dict. My mistake was in misunderstanding the `JSON` type to be any structure that would be ready to be serialized as JSON instead of an alias for `Mapping[str, AnyJSON]`.

Regarding PyCharm's warnings, please note I have not ignored your advice and have been aiming for warning free commits. In this particular case I am confused however as in my PyCharm I get no such warning as you describe, instead I see 9 weak warnings: 1 that another method in the file could be static, and 8 for exceeding line length which I fixed have fixed in the `1337-update-SCEA-portal-details` branch.

Screen Shot 2019-10-03 at 9 14 58 AM

I will investigate why my PyCharm doesn't show the same warning you experienced, and increase effort to pay closer attention to type hinting going forward.

**Comment by hannes-ucsc** [comment]
> Regarding PyCharm's warnings, please note I have not ignored your advice and have been aiming for warning free commits. In this particular case I am confused however as in my PyCharm I get no such warning as you describe, instead I see 9 weak warnings: 1 that another method in the file could be static, and 8 for exceeding line length which I fixed have fixed in the 1337-update-SCEA-portal-details branch.

@danielsotirhos I apologize. My accusation was unjustified.

**Pull request merged**